docs/source/en/model_doc/llava.md

```diff
@@ -116,7 +116,6 @@ For multiple turns conversation:
 
 [llava-1.5 models](https://huggingface.co/collections/llava-hf/llava-15-65f762d5b6941db5c2ba07e0) requires the following format:
 ```bash
-"USER: <image>\n<prompt> ASSISTANT:"
```

Comment:
a few lines were deleted where they shouldn't 🤔  I'm assuming it is because it has `pt>` in the deleted text?

(there are other cases in the diff)